README.md

```diff
@@ -266,3 +266,11 @@ a part of an existing mutable rule into a new mutable rule. In order for this ru
 the new rule must be identical to the phrasing of the clause or sentence in the old rule and the new rule must be
 self-contained, meaning it does not require a reference to the old rule to be understood. Similarly the old rule must
 still be self-contained and sensible with part of it removed.
+
+**319** Both players and non-players are allowed to submit a pull request to master on pimotte/nomic, 
+without this pull request containing any changes to any rule's text or the game state. 
+Such a pull request will be voted on as if they were a rule-change affecting only mutable rules. 
+Such a pull request will be merged by someone with the power to do so, in a timely fashion, if
+and only if the votes on the pull request are such that a rule-change with the same votes would
+be adopted. If a pull request only adds a name to the player list, that pull request
+will not be voted on. Instead, it may be merged by any person who has the right to do so.
```

Comment:
This means players leaving the game need permission (votes) from the other players.